mantidimaging/gui/windows/spectrum_viewer/model.py

```diff
@@ -370,10 +353,9 @@ def save_single_rits_spectrum(self, path: Path, error_mode: ErrorMode) -> None:
         Saves the spectrum for the RITS ROI to a RITS file.
 
         @param path: The path to save the CSV file to.
-        @param normalized: Whether to save the normalized spectrum.
-        @param error_mode: Which version (standard deviation or propagated) of the error to use in the RITS export
+        @param error_mode: Which version (standard deviation or propagated) of the error to use in the RITS export.
         """
-        self.save_rits_roi(path, error_mode, self.get_roi(ROI_RITS))
+        self.save_rits_roi(path, error_mode, self._roi_ranges["ROI_RITS"])
```

Comment:
This shouldn't be in quotes

`self.save_rits_roi(path, error_mode, self._roi_ranges[ROI_RITS])`

Otherwise when I do a RITS export I get:
```
2024-12-11 17:45:58,638 [mantidimaging.gui.windows.main.view:L562] ERROR: Traceback (most recent call last):
  File "/home/sam/git/mantidimaging/mantidimaging/gui/windows/spectrum_viewer/presenter.py", line 292, in handle_rits_export
    self.model.save_single_rits_spectrum(path, error_mode)
  File "/home/sam/git/mantidimaging/mantidimaging/gui/windows/spectrum_viewer/model.py", line 358, in save_single_rits_spectrum
    self.save_rits_roi(path, error_mode, self._roi_ranges["ROI_RITS"])
                                         ~~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'ROI_RITS'
```